src/displayapp/screens/Steps.cpp

```diff
@@ -51,19 +53,25 @@ Steps::Steps(Pinetime::Applications::DisplayApp* app,
   resetBtn = lv_btn_create(lv_scr_act(), nullptr);
   resetBtn->user_data = this;
   lv_obj_set_event_cb(resetBtn, lap_event_handler);
-  lv_obj_set_height(resetBtn, 50);
+  lv_obj_set_height(resetBtn, 30);
   lv_obj_set_width(resetBtn, 115);
   lv_obj_align(resetBtn, lv_scr_act(), LV_ALIGN_IN_BOTTOM_MID, 0, 0);
   resetButtonLabel = lv_label_create(resetBtn, nullptr);
   lv_label_set_text_static(resetButtonLabel, "Reset");
 
   currentTripSteps = motionController.GetTripSteps();
+  currentTripStepsDistance = motionController.GetTripSteps() * 0.0008;
```

Comment:
Why are `currentTripSteps` and `currentTripStepsDistance` being set here again, when they were just set above?

Reply:
I stumbled upon this in the code and honestly I had no idea. I thought it may be to refresh the tripsteps, so thats why I also added it again. I'll remove these two lines - the Refresh function can update it.